### PR TITLE
Visitor can choose language

### PR DESCRIPTION
**Pull request by perhegg**
#### Description
Changes proposed in this pull request:

* Adds i18n so visitor can choose between english and swedish

#### What I have learned working on this feature: 
I have learnt how to implement i18n.

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -8,3 +8,6 @@
     end
 end
 
+When("I click {string}") do |string|
```
Comment (magnus-thor) [inline features/step_definitions/basic_steps.rb]: Use a variable name that is more descriptive of the content

File: features/visitor_can_choose_language.feature [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Feature: Visitor can choose language
+
+As a Visitor
```
Comment (magnus-thor) [inline features/visitor_can_choose_language.feature]: Fix the indentation in this file

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: features/step_definitions/basic_steps.rb (hunk 1 above)
Comment (perhegg) [reply features/step_definitions/basic_steps.rb]: done

File: features/visitor_can_choose_language.feature (hunk 2 above)
Comment (perhegg) [reply features/visitor_can_choose_language.feature]: done

**Review by Joberg95 — APPROVED** [review]

**Review by NiciP — APPROVED** [review]

**Review by aleciabenjamin — APPROVED** [review]

File: app/views/articles/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,8 @@
+<%= link_to_unless_current "English", locale: 'en' %>
+<%= link_to_unless_current "Svenska", locale: 'sv' %>
+
+<%= link_to t('login'), "/" %>
```
Comment (magnus-thor) [inline app/views/articles/index.html.erb]: There is a login button in application layout file

File: app/views/articles/index.html.erb [hunk 4]
```diff
@@ -1,3 +1,8 @@
+<%= link_to_unless_current "English", locale: 'en' %>
```
Comment (magnus-thor) [inline app/views/articles/index.html.erb]: Is it only on the main page the user can switch between languages? if not move it to the application.html.erb file

File: app/views/articles/index.html.erb [hunk 5]
```diff
@@ -1,3 +1,8 @@
...
+<%= link_to_unless_current "Svenska", locale: 'sv' %>
+
+<%= link_to t('login'), "/" %>
+
 <h1> The Viking Times </h1>
```
Comment (magnus-thor) [inline app/views/articles/index.html.erb]: Will it be called The viking times in all languages? otherwise add a translation for it

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: app/views/articles/index.html.erb (hunk 3 above)
Comment (perhegg) [reply app/views/articles/index.html.erb]: removed

File: app/views/articles/index.html.erb (hunk 4 above)
Comment (perhegg) [reply app/views/articles/index.html.erb]: done

File: app/views/articles/index.html.erb (hunk 5 above)
Comment (perhegg) [reply app/views/articles/index.html.erb]: Yes!

File: features/visitor_can_choose_language.feature [hunk 6]
```diff
@@ -0,0 +1,17 @@
+Feature: Visitor can choose language
+
+    As a Visitor
+    In order to be able to read articles in my native language,
+    I would like to switch between languages English and Swedish
+
+Background:
```
Comment (magnus-thor) [inline features/visitor_can_choose_language.feature]: fix the indentation

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by magnus-thor — APPROVED** [review]
Good job

**Pull request merged**